CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
+## 10.2.3
+  - Fixed 8.x type removal compatibility issue to continue for monitoring endpoint from LogStash's X-Pack[#899](https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/899)
```

Comment:
I'm not sure this review comment accurately describes the change? I think the description given in #899 is more apt.

Reply:
Done, reworded a little bit. I hope it's not too long